scripts/ci/run-jenkins.sh

```diff
@@ -17,12 +17,12 @@ if [[ ${label} == w* ]] || [[ ${label} == 'debian-ppc64el' ]] || [[ ${label} ==
     # Override the defaults to skip profiles
     # only enable the mobile profiles and mobile_static on the main architectures
 		# only enable the concurrent collector by default on main unix archs
-    EXTRA_CONF_FLAGS="${EXTRA_CONF_FLAGS} --with-monodroid=no --with-monotouch=no --with-monotouch_watch=no --with-monotouch_tv=no --with-xammac=no"
+    EXTRA_CONF_FLAGS="${EXTRA_CONF_FLAGS} --with-monodroid=net_4_x"
```

Comment:
--with-monodroid=net_4_x doesn't make sense :)

Since this is the default now, we can just remove it.